ui/src/components/DrawCard.js

```diff
@@ -90,30 +125,42 @@ class DrawCard extends Component {
                     color={color}
                 />
             }
-            <NextRound  gameId={this.props.gameId} active={this.state.showNextRound}/>
+            <NextRound
+                active={this.state.showNextRound}
+                nextHandler={this.endRound}/>
         </div>
         );
     }
 }
 
-const Card = ({ card, doneHandler, drawHandler }) => (
+const Card = ({ card, showSkip, doneHandler, drawHandler }) => (
     <div className="card">
         <div className="card-value">
             <p>{card}</p>
         </div>
         <div className="actions">
             <button onClick={doneHandler} className="done">Got it!</button>
-            <button onClick={drawHandler} className="skip">Skip</button>
+            <button onClick={drawHandler}  disabled={!showSkip} className="skip">Skip</button>
```

Comment:
This is how I'm disable-ing the skip button when there is only one unread card left in the round. In the `drawCard` method on line 86 in this file I'm checking for the count of unused cards left in the round every time I draw a new random card:
```
                if (cardCount == 1) {
                    this.setState({ showSkip: false })
                }
```